tests/test_adversarial.py

```diff
@@ -77,43 +110,65 @@ def trainer(_algorithm_cls, _parallel: bool, tmpdir: str, _convert_dataset: bool
 
     gen_algo = util.init_rl(venv, verbose=1)
 
-    return _algorithm_cls(
+    trainer = _algorithm_cls(
         venv=venv,
         expert_data=expert_data,
+        expert_batch_size=expert_batch_size,
         gen_algo=gen_algo,
         log_dir=tmpdir,
     )
 
+    yield trainer
```

Comment:
Maybe close the `venv` explicitly at the end?

Reply:
Sounds good

Reply:
Ugh, actually it doesn't really matter for this case since it's just CartPole. Whatever, I'll add it anyways